public/docs/ts/latest/guide/architecture.jade

```diff
@@ -4,24 +4,24 @@ block includes
   - var _at_angular = '@angular'
 
 :marked
-  Angular 2 is a framework to help us build client applications in HTML and
+  Angular 2 is a framework for building client applications in HTML and
   either JavaScript or a language (like Dart or TypeScript) that compiles to JavaScript.
 
 block angular-parts
   :marked
-    The framework consists of several cooperating libraries, some of them core and some optional.
+    The framework consists of several libraries, some of them core and some optional.
 
 :marked
-  We write Angular applications by composing HTML *templates* with Angularized-markup,
+  You write Angular applications by composing HTML *templates* with Angularized markup,
   writing *component* classes to manage those templates, adding application logic in *services*,
   and boxing components and services in *modules*.
   
-  Then we launch the app by *bootstrapping* the top _root module_.
-  Angular takes over, presenting our application content in a browser and
+  Then you launch the app by *bootstrapping* the _root module_.
+  Angular takes over, presenting your application content in a browser and
   responding to user interactions according to the instructions we provided.
 
-  Of course there is more to it than this.
-  We'll learn the details when we dive into the guide chapters.
+  Of course, there is more to it than this.
+  You'll learn the details in the [*Tour of Heroes*](../tutorial) tutorial.
```

Comment:
Actually I think this is referring to the BASICS and DEVELOPER GUIDE pages, not the ToH, which precedes this page. Maybe just:

You'll learn the details in the pages that follow.